### PR TITLE
Add DeleteResultTemplate operation

### LINKED ISSUE #530: Add DeleteResultTemplate operation
Add a DeleteResultTemplate operation to support the deletion of a resultTemplate

### PR DESCRIPTION
**Pull request by EHJ-52n**
- fixes #530 Add DeleteResultTemplate operation
- minor adjustments to docker/docker-compose.yml

### CODE REVIEW

File: extensions/drt/coding/pom.xml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>api</artifactId>
+            <scope>test</scope>
+            <classifier>tests</classifier>
+        </dependency>
+    </dependencies>
+    <repositories>
```
Comment (CarstenHollmann) [inline extensions/drt/coding/pom.xml]: Why do you define the repositories?

File: extensions/drt/coding/src/test/resources/invalid_empty.xml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<drt:DeleteResultTemplate version="2.0.0" service="SOS" xmlns:drt="http://www.opengis.net/drt/1.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.opengis.net/drt/1.0 file://tsclient/RemminaDisk/home/eike/Code/git/common-xml/52n-xml-drt-v10/src/main/xsd/drt/1.0/DeleteResultTemplate.xsd">
```
Comment (CarstenHollmann) [inline extensions/drt/coding/src/test/resources/invalid_empty.xml]: SchemaLocation is a local path!

File: extensions/drt/coding/src/test/resources/invalid_mixed.xml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<drt:DeleteResultTemplate version="2.0.0" service="SOS" xmlns:drt="http://www.opengis.net/drt/1.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.opengis.net/drt/1.0 file://tsclient/RemminaDisk/home/eike/Code/git/common-xml/52n-xml-drt-v10/src/main/xsd/drt/1.0/DeleteResultTemplate.xsd">
```
Comment (CarstenHollmann) [inline extensions/drt/coding/src/test/resources/invalid_mixed.xml]: SchemaLocation is a local path!

File: extensions/drt/coding/src/test/resources/valid_offering_property_pairs.xml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<drt:DeleteResultTemplate version="2.0.0" service="SOS" xmlns:drt="http://www.opengis.net/drt/1.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.opengis.net/drt/1.0 file://tsclient/RemminaDisk/home/eike/Code/git/common-xml/52n-xml-drt-v10/src/main/xsd/drt/1.0/DeleteResultTemplate.xsd">
```
Comment (CarstenHollmann) [inline extensions/drt/coding/src/test/resources/valid_offering_property_pairs.xml]: SchemaLocation is a local path!

File: extensions/drt/coding/src/test/resources/valid_offering_property_pairs_working.xml [hunk 5]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<drt:DeleteResultTemplate version="2.0.0" service="SOS" xmlns:drt="http://www.opengis.net/drt/1.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.opengis.net/drt/1.0 file://tsclient/RemminaDisk/home/eike/Code/git/common-xml/52n-xml-drt-v10/src/main/xsd/drt/1.0/DeleteResultTemplate.xsd">
```
Comment (CarstenHollmann) [inline extensions/drt/coding/src/test/resources/valid_offering_property_pairs_working.xml]: SchemaLocation is a local path!

File: extensions/drt/coding/src/test/resources/valid_template_list.xml [hunk 6]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<drt:DeleteResultTemplate version="2.0.0" service="SOS" xmlns:drt="http://www.opengis.net/drt/1.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.opengis.net/drt/1.0 file://tsclient/RemminaDisk/home/eike/Code/git/common-xml/52n-xml-drt-v10/src/main/xsd/drt/1.0/DeleteResultTemplate.xsd">
```
Comment (CarstenHollmann) [inline extensions/drt/coding/src/test/resources/valid_template_list.xml]: SchemaLocation is a local path!

File: extensions/drt/coding/src/test/resources/valid_offering_property_pairs.xml [hunk 7]
```diff
@@ -0,0 +1,7 @@
...
+	<drt:offering>offering1</drt:offering>
```
Comment (CarstenHollmann) [inline extensions/drt/coding/src/test/resources/valid_offering_property_pairs.xml]: I would suggest to define a subelement that contains a offering/observedProperty pair.

**Comment by EHJ-52n** [comment]
@CarstenHollmann 

I have updated the PR, so please re-review.

**Pull request merged**